utils/build-presets.ini

```diff
@@ -2152,7 +2152,7 @@ test-installable-package
 reconfigure
 
 swift-install-components=compiler;clang-resource-dir-symlink;stdlib;sdk-overlay;parser-lib;toolchain-tools;license;sourcekit-xpc-service;swift-remote-mirror;swift-remote-mirror-headers
-llvm-install-components=llvm-cov;llvm-profdata;IndexStore;clang;clang-headers;compiler-rt;clangd
+llvm-install-components=llvm-cov;llvm-profdata;IndexStore;clang;clang-resource-headers;compiler-rt;clangd
```

Comment:
Note: it would be much nicer to avoid these issues by reusing the `mixin_osx_package_base` preset for `tensorflow_osx_base` instead of duplicating it.

Here's a historical note for the `tensorflow_osx_base` preset:

```
# Note: tensorflow_osx_base does not mix in "buildbot_osx_package" or
# "mixin_osx_package_base" because they enable ios/tvos/watchos and tests by
# default.
# Instead, options are copied here individually.
[preset: tensorflow_osx_base]
mixin-preset=mixin_lightweight_assertions
```

We could try mixing in `mixin_osx_package_base` for `tensorflow_osx_base` and overriding `ios`/`tvos`/`watchos` with `skip-ios`/`skip-tvos`/`skip-watchos`.

Reply:
Filed TF-941 to track this issue.